services/common/src/components/tailings/BasicInformation.tsx

```diff
@@ -36,9 +36,51 @@ interface BasicInformationProps {
   isEditMode: boolean;
 }
 
+// Provide a mapping of the field names to the title and data for the field
+// This is used to display the field names and values in a more user-friendly way in the history modal
+const historyDiffValueMapper = {
```

Comment:
For the purposes of these value mappers, will we need to always provide an exhaustive list of all fields we want tracked?  I'm just thinking about what will happen if the data changes and updates to these get missed.

Reply:
I set it up to default to using the values as is if missing from the mapper, and just transforming the title from snake_case to Title Case - I'll update the comments to make that more clear